<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProfileListWindow</class>
 <widget class="QMainWindow" name="ProfileListWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>378</width>
    <height>429</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Список профилей</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/resources/logo.ico</normaloff>:/resources/logo.ico</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
   background-color: #13171c;
   color: #f5f5f5;
   font-family:  Rubik;
}

QLineEdit {
	border: 2px solid #cfcfcf;
	border-radius: 4px;
}

QPushButton {
    background-color: #1d2024;
	border: none;
 	border-collapse: separate;
	border-radius: 5px;
}

QPushButton::hover {
    background-color: #222a33;
}

QPushButton::pressed {
    background-color: #1d2329;
}

QListWidget {
	border: 2px solid #cfcfcf;
	border-radius: 7px;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QListWidget" name="listWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>361</width>
      <height>321</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QWidget {
   background-color: #586675;
}

QPushButton::hover {
   background-color: #505d6b;
}

QPushButton::pressed {
   background-color: #44515e;
}</string>
    </property>
   </widget>
   <widget class="QPushButton" name="editProfileButton">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>380</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Rubik</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Редактировать</string>
    </property>
   </widget>
   <widget class="QLabel" name="title">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Rubik</family>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
	color: rgb(220, 220, 220)
}</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Редактирование профилей&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="minimizeButton">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>10</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Rubik</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>_</string>
    </property>
   </widget>
   <widget class="QPushButton" name="closeButton">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>10</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Rubik</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>X</string>
    </property>
   </widget>
  </widget>
  <action name="action">
   <property name="text">
    <string>Сбросить базу данных</string>
   </property>
  </action>
  <action name="action_2">
   <property name="text">
    <string>Выйти</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
